packages/nextjs-template/playwright.config.ts

```diff
@@ -1,7 +1,11 @@
+import path from "path";
 import { PlaywrightTestConfig, devices } from "@playwright/test";
 
+const testDir = path.join(__dirname, "e2e");
```

Comment:
we currently treat tests as e2e tests (eg. a lot of our plugin tests are integration tests). but for playwright, since its a bit heavier lift, i'm fine keeping it in e2e